scripts/src/main/resources/scripts/command/terraform

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup help"
+  fi
+  exit
+fi
+
+if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+
+# Check if terraform is installed.
+function doIsTerraformInstalled() {
+  if command -v terraform &> /dev/null
+  then 
+    return
+  else
+    return 255
+  fi
+}
+
+# Call Terraform with specified arguments.
+function doRun() {
+  doSetup silent
+  terraform "${@}"
+}
+
+function doSetup() {
+
+  if doIsTerraformInstalled 
+  then
+    if [ "${1}" != "silent" ] && ! doIsQuiet
+    then 
+      terraform version
+      echo
+    fi 
+  else
+    # Check if TERRAFORM_HOME is already set
+    if [ -z "${TERRAFORM_HOME}" ]
+    then
+      doEcho "set TERRAFORM_HOME..."
```

Comment:
I would remove this `doEcho "set TERRAFORM_HOME..."`

Reply:
deleted